parent/pom.xml

```diff
@@ -74,8 +74,8 @@
     <version.jakarta-ee-spec>10.0.0</version.jakarta-ee-spec>
 
     <version.eirslett.frontend.maven.plugin>1.12.1</version.eirslett.frontend.maven.plugin>
-    <version.nodejs>17.9.1</version.nodejs>
-    <version.npm>8.11.0</version.npm>
+    <version.nodejs>20.14.0</version.nodejs>
+    <version.npm>10.7.0</version.npm>
```

Comment:
❓ CentOS 7 doesn't support a higher version > 17.9.1. While we might migrate all web apps building stages in "one-off", how do we handle this in our release CI?

Reply:
Good point. We could introduce and activate a Maven profile for the release pipeline only.

Reply:
That would be an interim solution until we migrate from the Jenkins freestyle jobs. However, we might need to bump NodeJS in the future since newer versions of our third-party libraries might not be compatible with the old version in the future.
Alternatively, can we use the Maven Image in the release jobs already?

Reply:
https://github.com/camunda/team-infrastructure/issues/641